figure.html

```diff
@@ -852,15 +836,63 @@ <h2>
 
 </li>
 
-<h2><a class="anchor" id="saving"></a>Configuring, saving and exporting your
-    figure
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<h2><a class="anchor" id="paper_setup"></a>
+    Paper Setup
 </h2>
 
+<li>
+    <p class="first">
+        To change the paper color, size and page count, click on the
+        <span class="bold">File</span> drop-down menu.
+    </p>
+
+    <p>Select <span class="bold">Paper Setup ...</span>.</p>
+
+    <p>Click on the <span class="bold">Page Color</span> widget to open a Color-picker.</p>
+
+    <p>Select the number of pages from the drop-down.</p>
+
+    <p>Select the size and orientation of your canvas.</p>
+
+    <div class="line"><br /></div>
+
+    <img alt="../images/figureSize.jpg" class="align-center" src="images/figureSize.jpg" style="width: 100%;" />
```

Comment:
we could try to unify the image type mix now of jpg and png

Reply:
OK. Most are jpg, so we should probably unify to that.
The only reason that the pngs are there is that they're unmodified screenshots. But I can convert them in Photoshop.